### PR TITLE
MLH-492 add regression tests for name

### PR DESCRIPTION
**Pull request by Timuriagudin**

### CODE REVIEW

File: test/regression/fieldName.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+describe('Error "Required" appears after input field filled and cleared', function () {
+
+    it('TC-013 Input Lady555 in Hero\'s name field', function () {
+        browser.url('');
+        const name1 = $(sel.name).setValue(data.name);
+        $(sel.name).doubleClick();
+        browser.keys("Delete");
+        const text = $(sel.explain).getProperty('innerText');
+        expect(text).toEqual(exp.required);
+
```
Comment (vasylynash) [inline test/regression/fieldName.js:14]: Remove empty line

File: test/regression/fieldName.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+});
+
+describe('Error "70 symbols max" appears after input more than 70 symbols', function () {
+
+    it('TC-017 Input 71 symbols in Hero\'s name field', function () {
+        browser.url('');
+        const name1 = $(sel.name).setValue(data.name71);
+        const text = $(sel.explain).getProperty('innerText');
+        expect(text).toEqual(exp.error);
+
```
Comment (vasylynash) [inline test/regression/fieldName.js:25]: Remove empty line

File: wdio.conf.js [hunk 3]
```diff
@@ -18,11 +18,14 @@ exports.config = {
     //
     specs: [
         './test/smoke/storyPage.js',
```
Comment (vasylynash) [inline wdio.conf.js:20]: Remove the list of the files

File: wdio.conf.js [hunk 4]
```diff
@@ -18,11 +18,14 @@ exports.config = {
...
-        './test/regression/createButton.js'
+        './test/regression/createButton.js',
+        './test/regression/fieldName.js'
     ],
     // Patterns to exclude.
     exclude: [
-        // 'path/to/excluded/files'
+        './test/smoke/storyPage.js',
```
Comment (vasylynash) [inline wdio.conf.js:26]: Remove the list of the files

**Review by vasylynash — CHANGES_REQUESTED** [review]

File: wdio.conf.js (hunk 4 above)
Comment (Timuriagudin) [reply wdio.conf.js:26]: should I leave only  // 'path/to/excluded/files' ?

File: wdio.conf.js (hunk 3 above)
Comment (Timuriagudin) [reply wdio.conf.js:20]: should I leave only './test/smoke/storyPage.js', ?

File: test/regression/fieldName.js [hunk 5]
```diff
@@ -11,7 +11,6 @@ describe('Error "Required" appears after input field filled and cleared', functi
         browser.keys("Delete");
         const text = $(sel.explain).getProperty('innerText');
         expect(text).toEqual(exp.required);
```
Comment (vasylynash) [inline test/regression/fieldName.js:13]: toEqual is not from Chaijs (change all instances)

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by Anastasiia-Bandurova — APPROVED** [review]

**Review by vasylynash — APPROVED** [review]

**Pull request merged**